lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -437,6 +437,8 @@ namespace Js
         library->AddMember(intlNativeInterfaces, Js::PropertyIds::winglob, library->GetFalse());
 #endif
 
+        library->AddMember(intlNativeInterfaces, PropertyIds::showPlatform, TO_JSBOOL(scriptContext, CONFIG_FLAG(IntlPlatform)));
```

Comment:
Only under DBG?

Reply:
I used FLAGNR which, as far as I can tell, means the flag is only available in debug and test builds... I suppose I could add another check here but since platform is fully internal in release, I am not sure it matters too much? I guess theres some minimal savings of an extra property on the object at runtime.

Reply:
You can add DebugOnly on the beginning of the line. No point to add the property when we don't have to.